packages/opencensus-core/src/exporters/console-exporter.ts

```diff
@@ -74,3 +71,29 @@ export class ConsoleExporter implements types.Exporter {
     });
   }
 }
+
+export class ConsoleStatsExporter implements types.StatsExporter {
```

Comment:
For each class that you introduce, can you add docs describing its purpose and role?

Reply:
Done.